src/main/java/edu/harvard/iq/dataverse/ingest/plugin/spi/PartiallyOrderedSet.java

```diff
@@ -78,27 +78,29 @@
  * supplied by a trusted vendor over those supplied by another.
  *
  */
-class PartiallyOrderedSet extends AbstractSet {
+class PartiallyOrderedSet<E> extends AbstractSet<E> {
```

Comment:
A look over of this set would also be a good idea. I'm 95% sure I got the generics right, but more eyes make better code.